sites/public/styles/overrides.scss

```diff
@@ -1,5 +1,53 @@
 /* Overrides the default ui-components styles with Detroit-specific styles. */
-@import url("https://fonts.googleapis.com/css?family=Montserrat:400,600,700");
+@import "../public/static/fonts/Montserrat.css";
+
+@import "@bloom-housing/ui-components/src/global/tokens/fonts.scss";
+@import "@bloom-housing/ui-components/src/global/tokens/borders.scss";
+@import "@bloom-housing/ui-components/src/global/tokens/colors.scss";
+@import "@bloom-housing/ui-components/src/global/tokens/screens.scss";
+@import "@bloom-housing/ui-components/src/global/tokens/sizes.scss";
+@import "@bloom-housing/ui-components/src/global/mixins.scss";
+@import "@bloom-housing/ui-components/src/global/text.scss";
+@import "@bloom-housing/ui-components/src/global/markdown.scss";
+@import "@bloom-housing/ui-components/src/global/lists.scss";
+@import "@bloom-housing/ui-components/src/global/accordion.scss";
+
+:root {
+  // Notes
+  // Removed the tailwind config overrides but it already wasn't working so that wasn't the problem
+  // It doesn't like when they're in an array
+  // Some of these work, and some of them don't? The colors seem to work, the fonts not so much
+  // But the font IS getting applied if I use a built-in font so it seems like Montserrat is just not loading - aka if I override them both with just "Verdana"
+  // But nothing that I can tell so far was removed that would allow Montserrat to work
+  // The configs are the same
+  // The font style exists correctly in the browser / it's using the detroit-etc flavor
+  // The UIC fonts.scss import at the top would technically override these if it was put in the wrong place but the right one is in the browser so it shouldn't matter
+  // If I look in the network tab for fonts, prod is fetching it and dev/local build are not
```

Comment:
Is this whole comment still needed? As far as I could tell the Monterrat is working now? So is this just your working notes?